CO2_Gadget_TFT.h

```diff
@@ -655,6 +655,7 @@ void displayShowValues(bool forceRedraw = false) {
     if (forceRedraw) {
         // Serial.println("-->[TFT ] Displaying values. Force Redraw: " + String(forceRedraw ? "true" : "false"));
         tft.fillScreen(TFT_BLACK);  // Remove previous remains in the screen
+        spr.fillSprite(TFT_BLACK);  // Remove previous remains in the screen
```

Comment:
I'm unsure why this is included here since no sprites are created at this point. Could this commit be an error? It seems unrelated to this PR regardless.

Reply:
In S3 version, display is not updated when temp/hum/battery is hidden until menu in and out and display refreshed completely. 
Showing a hidden value is updated well and quickly. 
This line is a try to repair this minor bug.